certbot-nginx/certbot_nginx/configurator.py

```diff
@@ -58,6 +58,8 @@ class NginxConfigurator(common.Plugin):
 
     hidden = True
 
+    DEFAULT_PORT = '80'
```

Comment:
I didn't realize that in Nginx, 80 was literally the default port in that the lack of a port number defaults to 80. In that case, you can ignore this comment if you want.